recipes/_ghost.rb

```diff
@@ -4,6 +4,10 @@
     not_if { ::File.exist?("#{Chef::Config[:file_cache_path]}/ghost.zip") }
 end
 
+directory node['ghost-blog']['install_dir'] do
```

Comment:
This breaks the conditional on https://github.com/gsaslis/chef-ghost/blob/1e58310a0e7b949739aa84e7b0c82cc6de40ae46/recipes/_ghost.rb#L14. Any particular reason you want to create the directory before the `unzip` does?